core/arch/arm/plat-mediatek/pm_debug.h

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
If we are using pm_stubs I'd guess that we could remove this file and also remove it where it has been included?